### PR TITLE
github action for relaychain genesis files

### PR DESCRIPTION
**Pull request by Garandor**
Signed-off-by: Adam Reif <Garandor@manta.network>

## Description

---

Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
- [x] Connected to an issue with discussion and accepted design using zenhub "Connect issue" button below
- [x] Added **one** label out of the `L-` group to this PR
- [x] Added **one or more** labels from the `A-` and `C-` groups to this PR
- [x] Explicitly labelled `A-calamari`, `A-dolphin` and/or `A-manta` if your changes are meant for/impact either of these (CI depends on it)
- [x] Re-reviewed `Files changed` in the Github PR explorer.

Situational Notes:
- If adding functionality, write unit tests!
- If importing a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- If needed, update our Javascript/Typescript APIs. These APIs are officially used by exchanges or community developers.
- If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inherited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.

### CODE REVIEW

File: .github/workflows/genesis_files_for_relay.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        run: |
+          $HOME/manta build-spec --chain $CHAIN --raw > ./$CHAIN-genesis.json
+          sha256sum $CHAIN-genesis.json | tee $CHAIN-genesis.json.sha256
+      - name: Export genesis state
+        run: |
+          $HOME/manta export-genesis-state --chain $CHAIN > $CHAIN-genesis.state
+          sha256sum $CHAIN-genesis.state | tee $CHAIN-genesis.state.sha256
+      - name: Export genesis wasm
+        run: |
+          $HOME/manta export-genesis-state --chain $CHAIN > $CHAIN-genesis.wasm
```
Comment (ghzlatarev) [inline .github/workflows/genesis_files_for_relay.yml]: export wasm not state here, also I've always used --raw

Comment (Garandor) [reply .github/workflows/genesis_files_for_relay.yml]: thx, fixed the copypasta

according to https://docs.substrate.io/tutorials/connect-relay-and-parachains/acquire-a-testnet-slot/#export-required-files
these don't need to be `--raw`. I'd keep em that way, they did get registered successfully by shumo the last time around without this flag too

Comment (ghzlatarev) [reply .github/workflows/genesis_files_for_relay.yml]: Ok lemme just double check this doc, i'm not even sure what the flag did.

Comment (ghzlatarev) [reply .github/workflows/genesis_files_for_relay.yml]: So the output is going to be hex instead of binary, so that should be fine.

**Review by ghzlatarev — DISMISSED** [review]

**Review by ferrell-code — DISMISSED** [review]

**Review by ferrell-code — APPROVED** [review]

**Review by ghzlatarev — APPROVED** [review]

**Pull request merged**